miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditLibraryController.java

```diff
@@ -686,12 +700,21 @@ protected LibraryDto createDtoFromParent(Sample item) {
 
     @Override
     protected void writeConfiguration(ObjectMapper mapper, ObjectNode config) {
-      writeLibraryConfiguration(mapper, config);
+      additionalConfigFunction.accept(mapper, config);
       if (templatesByProjectId != null && !templatesByProjectId.isEmpty()) {
         config.putPOJO(Config.TEMPLATES, templatesByProjectId);
       }
+      if (sort != null) {
+        config.put(Config.SORT, sort);
+      }
     }
-  };
+
+    public ModelAndView propagate(String idString, int replicates, String sort, ModelMap model) throws IOException {
+      this.sort = sort;
+      return propagate(idString, replicates, model);
+    }
+
+  }
 
   private void writeLibraryConfiguration(ObjectMapper mapper, ObjectNode config) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "mapper". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)